### DISCUSSION TITLE
Set a custom numbering scheme for nested ordered lists

### DISCUSSION THREAD

**Discussion by deighvebee** [GitbookIO/community#202] [Feature Requests]
### What's your feature/idea?

Nested ordered lists (aka substeps) should not use the same numbering scheme as the root-level steps. For example:
![image](https://github.com/GitbookIO/community/assets/16226947/44752ed9-6e9a-4a72-b855-63dd43e37520)

### What's your desired outcome?

**Improved Default Behavior**
By default, I would expect nested numbering to be something like:
Arabic numeral (1) -> lowercase alpha (a) -> lowercase roman numeral (i)

**Configurable Numbering Schemes**
Allow the user to specify the numbering scheme for each ordered list.

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

### Anything else?

I'm rating this a medium because it's preventing me from using substeps. Not the biggest deal, but this comes up in documentation a lot.

**Comment by taszap** [comment]
Hello,

Thanks for your feedback!
Marking this as a related to #17
